### PR TITLE
Express integration

### PR DESCRIPTION
**Pull request by itamarst**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,8 +35,15 @@ virtualenv3:
 python3-dependencies: virtualenv3
 	virtualenv3/bin/pip install -r dev-requirements.txt
 
+node_modules:
+	mkdir node_modules
```
Comment (bozzzzo) [inline Makefile:40]: Add a `rm -fr node_modules` to the clean target

File: javascript/datawire_mdk_express/.bumpversion.cfg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[bumpversion]
+current_version = 1.0.0
+commit = True
+tag = True
+tag_name = v{new_version}
```
Comment (bozzzzo) [inline javascript/datawire_mdk_express/.bumpversion.cfg]: Will this create a mess with tags in the mdk repo? how will we distinguish tags for mdk and datawire_mdk_express?

Comment (itamarst) [reply javascript/datawire_mdk_express/.bumpversion.cfg]: Oops. It won't. I meant to change it so `commit` and `tag` were `False`.

File: Makefile (hunk 1 above)
Comment (itamarst) [reply Makefile:40]: Done.

File: javascript/datawire_mdk_express/.bumpversion.cfg (hunk 2 above)
Comment (itamarst) [reply javascript/datawire_mdk_express/.bumpversion.cfg]: Got rid of extra version thing.

**Review by bozzzzo — APPROVED** [review]

**Pull request merged**